### DISCUSSION TITLE
Deploy getStaticPaths

### DISCUSSION THREAD

**Discussion by rserafim** [vercel/next.js#25424] [Help] [answered]
Example:
I have a total of 100

Using getStaticPaths to generate a total of 30 pages at build time. In addition to the falback: true property.

So in the deploy I will already have 30 pages generated statically.

And with each new request, the others are generated, thus completing all of them (100)

If I perform a new deploy, with 30 pages in the build time again, will the other 70 that were generated by new accesses still exist? Or with each new deployment they get lost?

**Comment by bobarros** [comment]
You shouldn't lose them: https://vercel.com/docs/next.js/incremental-static-regeneration#isr:-not-just-caching

**Comment by rserafim** [comment, marked as answer]
As I understand it, every new deploy will only generate those static pages defined at build time. And those that had already been generated will be created again.
